app/application/route.ts

```diff
@@ -1,26 +1,19 @@
+import { service } from '@ember-decorators/service';
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 import OSFAgnosticAuthRouteMixin from 'ember-osf-web/mixins/osf-agnostic-auth-route';
 
-export default class Application extends Route.extend(OSFAgnosticAuthRouteMixin, {
-    beforeModel(...args) {
+export default class Application extends Route.extend(OSFAgnosticAuthRouteMixin) {
+    @service moment;
+
+    beforeModel(this: Application, ...args) {
```

Comment:
@aaxelb I missed this one, but this._super breaks when moving down to the class body.

We should make it:

```TypeScript
    constructor() {
        super();
        this.get('moment').setTimeZone('UTC');
    }
```

Not sure why the timezone would need to be reset before every model, so constructor should work

Reply:
Yeah, constructor's a better place for that 👍 

...but do we actually want to display all times in UTC?